index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr. Nabeel</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.pixabay.com/photo/2015/01/08/18/29/entrepreneur-593358_960_720.jpg"
+          alt="Dr.Nabeel sitting"
+        />
+        <figcaption id="img-caption">we are good</figcaption>
```

Comment:
Please use **class selector** more often instead of using **id selector**😉.  According to HTML best practice it is adviced to use class selector more often and use id selector only in unique cases😊.